services/worker/pyproject.toml

```diff
@@ -58,6 +58,7 @@ flake8 = "^3.9.2"
 flake8-pep585 = "^0.1.7"
 flake8-unused-arguments = "^0.0.13"
 isort = "^5.12.0"
+moto = "^4.2.5"
```

Comment:
(maybe) install in the dev dependencies of libcommon instead

Reply:
(feel free to ignore)

Reply:
I think I will leave it as it is now, anyway once we use `fsspec` instead we will probably remove all the references to boto3 and moto3.